tests/python/cpu/test_mklml.py

```diff
@@ -0,0 +1,52 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+MKLML related test cases
+"""
+
+import logging
+import os
+
+def test_mklml_install():
+    """
+    This function will check if MXNet is built/installed correctly
+    when compiling with Intel MKLML library, the method is try
+    to import mxnet module and see if correct mklml library is
+    mapped to this process's address space
+    """
+    logging.basicConfig(level=logging.INFO)
```

Comment:
Will this change the global logging level?  Would this be a problem if say a user wanted to run the entire test suit in warning only mode?